### PR TITLE
Select Orchard notes in FindSpendableInputs

### PR DESCRIPTION
**Pull request by nuttycom**

### CODE REVIEW

**Review by str4d — APPROVED** [review]
utACK; this should enable detection of funds once #5596 merges. Could use a unit test (using the C++ `orchard::Builder` directly).

**Comment by nuttycom** [comment]
The build failures here are transient CI bs.

File: src/wallet/wallet.cpp [hunk 1]
```diff
@@ -1954,6 +1954,39 @@ SpendableInputs CWallet::FindSpendableInputs(
...
+                }
+            }
+            return {};
+        },
+        [&](const auto& addr) -> std::vector<OrchardIncomingViewingKey> { return {}; }
+    }, selector.GetPattern());
+
+    for (const auto& ivk : orchardIvks) {
+        // TODO ORCHARD: Allow the minimum number of confirmations
+        // to be specified
```
Comment (ebfull) [inline src/wallet/wallet.cpp:1981]: Is there an issue open for this?

**Review by ebfull — APPROVED** [review]

File: src/wallet/wallet.cpp (hunk 1 above)
Comment (nuttycom) [reply src/wallet/wallet.cpp:1981]: https://github.com/zcash/zcash/issues/5599

**Pull request merged**